src/components/dialog/dialog.js

```diff
@@ -114,6 +115,16 @@ const VUIDialog = ({
         dialog.setAttribute('tabindex', 1);
         dialog.setAttribute('aria-hidden', false);
 
+        // Alerts don't have close buttons
+        if (hasCloseBtn && !isAlert) {
+            const closeBtnEl = createEl({
+                element: 'button',
+                className: 'js-dialog-btn-close',
+                type: 'button'
+            })
```

Comment:
Yeah definitely. The text node option for createEl is in the PR for that util so I'll have to wait until that goes in before I can use it here